terraform/variables.tf

```diff
@@ -17,10 +17,21 @@ variable "vpc_cidr" {
   description = "cidr block for VPC"
 }
 
-# networks which are allowed to talk with the k8s API
+variable "eks_subnet_count" {
+  default     = 2
+  description = "number of subnets used for EKS"
+}
+
+variable "service_subnet_count" {
+  default     = 2
+  description = "number of subnets used for RDS and other services"
+}
+
 variable "kubecontrolnets" {
```

Comment:
Per our slack chat, this can be removed, along with the rule that references it. I've verified locally that I didn't need it.

Reply:
Awesome.  Thanks for spotting that!  Removed!